docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -461,6 +461,13 @@ kubeadm reset
 If you wish to start over simply run `kubeadm init` or `kubeadm join` with the
 appropriate arguments.
 
+**Note**: `kubeadm reset` will not delete any etcd data. This means that if you run 
```

Comment:
**if external etcd is used**. In case of local etcd, everything's wiped out.

Reply:
good catch